ethereum/p2p/src/main/java/org/hyperledger/besu/ethereum/p2p/rlpx/connections/netty/DeFramer.java

```diff
@@ -105,8 +110,11 @@ protected void decode(final ChannelHandlerContext ctx, final ByteBuf in, final L
     while ((message = framer.deframe(in)) != null) {
 
       if (hellosExchanged) {
+
         out.add(message);
+
       } else if (message.getCode() == WireMessageCodes.HELLO) {
+
```

Comment:
these blank lines seem unnecessary

Reply:
I just put these empty lines in to make it a bit simpler to find where the different messages are looked at. If you think that it does not help I can remove these blank lines.